sdk/textanalytics/ai-text-analytics/test/README.md

```diff
@@ -0,0 +1,11 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The environment variables you will need to properly assign to run the live test of this project are the following:
+
+- `TEXT_ANALYTICS_API_KEY`: The primary API key of your Azure Text Analytics account.
+- `TEXT_ANALYTICS_API_KEY_ALT` (optional): The secondary API key of your Azure Text Analytics account.
+- `ENDPOINT`: The endpoint of your Azure Text Analytics account.
```

Comment:
@willmtemple, This is a very badly named env variable. Can we have the `TEXT_ANALYTICS` prefix?

Reply:
I will make a separate PR for that!

Reply:
I made this issue first: https://github.com/Azure/azure-sdk-for-js/issues/8463